static/static.go

```diff
@@ -1,4 +1,2 @@
-// Package static embeds static (JS, HTML) resources right into the binaries
+// Package static contains JSON config files and keys
```

Comment:
I wonder if package static should be renamed to something else!

Reply:
Do we still need this file? There are no other Go files so I believe we can get rid of this one as well.

Reply:
Earlier it gave me an error something like `static does not have any go files`, but I tried just now and It worked fine even after removing this file. So, removing this in next commit.